sorting/bubbleSort.js

```diff
@@ -0,0 +1,21 @@
+const bubbleSort = (arr) => {
+
+    if(arr !== null) {
+        let len = arr.length;
+
+        let temp = 0;
```

Comment:
why do you need temp here? if this is only being used onwards line 7 and not used for retaining any value